modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java

```diff
@@ -43,6 +43,24 @@
  *
  */
 public class ConsistencyCommand extends AbstractCommand<Object> {
+    /** Cache. */
+    public static final String C = "-c";
```

Comment:
As I can see, Ignite doesn't support shortened params for any commands in the control utility. Some operations use the full "--cache" param, but no one uses the "-c" param. Let's do it similarly.

Reply:
Why not? 
This is useful! Let me be the first! :)

Reply:
IMHO, it's useful for commands that run frequently in console and frequently used with combination of multiple options, like `tar -cvf`, or `ls -lrt`, or `grep -HIi`.

Reply:
It's really harder to ake type at `P` than at `PARTITION`. 
Also, the command is shorter, which is important.